markdown/ch2.markdown

```diff
@@ -83,13 +83,13 @@ Together, these two functions illustrate many of the things that make NumPy powe
 also among the most useful. We will explore it further in the text that
 follows.
 
-## Getting the data
+## Getting the Data
 
-As in Chapter 1, we will be working with the The Cancer Genome Atlas (TCGA) skin cancer RNAseq data set.
+As in Chapter 1, we will be working with the The Cancer Genome Atlas (TCGA) skin cancer RNAseq dataset.
 Our goal is to predict mortality in skin cancer patients using their RNA expression data.
-By the end of this chapter we will have reproduced a simplified version of [Figures 5A and 5B](http://www.cell.com/action/showImagesData?pii=S0092-8674%2815%2900634-0) of a [paper](http://dx.doi.org/10.1016/j.cell.2015.05.044) from the TCGA consortium.
+As mentioned earlier, by the end of this chapter we will have reproduced a simplified version of [Figures 5A and 5B](http://www.cell.com/action/showImagesData?pii=S0092-8674%2815%2900634-0) of a [paper](http://dx.doi.org/10.1016/j.cell.2015.05.044) from the TCGA consortium.
```

Comment:
Did we?

Reply:
yes we did. You have to click on Fig 5 from the link (no way to direct link, hooray for the modern internet!)